app/ReactAdmin.js

```diff
@@ -44,7 +44,7 @@ class ReactAdmin extends React.Component {
         Pace.stop();
     }
 
-    componentWillMount() {
+    componentWillReceiveProps() {
```

Comment:
Mandatory because `Router.run` should be executed after routing configuration in `config.js`.